@kgerheiser there is still a problem with bufr:
```
[ 28%] Building Fortran object src/CMakeFiles/bufr_8_f.dir/atrcpt.f.o
cd /global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/build/src && /global/panfs01/admin/opt/intel/oneAPI/2021.4.0.3347/compiler/2
021.4.0/linux/bin/intel64/ifort -DLITTLE_ENDIAN -DSTATIC_ALLOCATION -I/global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/build/src
 -I/global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/src -fPIC -w -g -traceback -O3 -module include/bufr_8 -fPIC -r8 -i8 -c /glob
al/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/src/atrcpt.f -o CMakeFiles/bufr_8_f.dir/atrcpt.f.o
[ 28%] Building Fortran object src/CMakeFiles/bufr_8_DA_f.dir/mrginv.f.o
curl: (6) Could not resolve host: ftp.emc.ncep.noaa.gov
cd /global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/build/src && /global/panfs01/admin/opt/intel/oneAPI/2021.4.0.3347/compiler/2
021.4.0/linux/bin/intel64/ifort -DDYNAMIC_ALLOCATION -DLITTLE_ENDIAN -I/global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/build/sr
c -I/global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/src -fPIC -w -g -traceback -O3 -module include/bufr_8_DA -fPIC -r8 -i8 -c /
global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/src/mrginv.f -o CMakeFiles/bufr_8_DA_f.dir/mrginv.f.o
make[2]: *** [test/CMakeFiles/get_bufr_test_data.dir/build.make:80: test/bufr.tar] Error 6
make[2]: Leaving directory '/global/panfs/users/Xdheinz/hpc-stack/hpc-stack-20211102/pkg/bufr-bufr_v11.5.0/build'
make[1]: *** [CMakeFiles/Makefile2:1416: test/CMakeFiles/get_bufr_test_data.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 28%] Building Fortran object src/CMakeFiles/bufr_4_f.dir/datelen.f.o
```
Note the `curl: (6) Could not resolve host: ftp.emc.ncep.noaa.gov`. I thought I had seen conversations about this in the past, but I am not sure.

--- a/libs/build_nceplibs.sh
+++ b/libs/build_nceplibs.sh
@@ -234,7 +234,12 @@ case $name in
     ;;
   bufr)
     if [[ ${using_python:-} =~ [yYtT] ]]; then
-      extraCMakeFlags="-DENABLE_PYTHON=ON"
+      extraCMakeFlags="-DENABLE_PYTHON=ON "
+    fi
+    if [[ $MAKE_CHECK =~ [yYtT] ]]; then
+        extraCMakeFlags+="-DBUILD_TESTS=ON"
+    else
+        extraCMakeFlags+="-DBUILD_TETS=OFF"
     fi
     ;;
   nemsio)

